targets/simple_switch_grpc/switch_runner.h

```diff
@@ -49,9 +49,9 @@ class SimpleSwitchGrpcRunner {
   // there is no real need for a singleton here, except for the fact that we use
   // PIGrpcServerRunAddr, ... which uses static state
   static SimpleSwitchGrpcRunner &get_instance(
-      int max_port = 512, bool enable_swap = false,
+      bm::DevMgrIface::port_t max_port = 512, bool enable_swap = false,
       std::string grpc_server_addr = "0.0.0.0:50051",
-      int cpu_port = -1,
+      bm::DevMgrIface::port_t cpu_port = -1,
```

Comment:
You kept -1 as the default value here, but changed it to 0 below?

Reply:
I missed this one. Fixed.